### PR TITLE
#13 updated styles watch task to contain propper path

### PR DESCRIPTION
**Pull request by blakejwc**
Added the change suggested by Kent and updated the package.json urls.

### CODE REVIEW

**Comment by blakejwc** [comment]
I guess this will also address #7

File: package.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
   "description": "Gulp build process tasks that provide watchify and sass compilation.",
   "repository": {
     "type": "git",
-    "url": "https://github.com/arikwex/quick-sip"
+    "url": "https://github.com/vecnatechnologies/quick-sip"
```
Comment (mandragorn) [inline package.json]: This also resolves #7

File: tasks/watch.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ module.exports = function(gulp, options) {
 
     if (!options.styles.skip) {
       buildTasks.push(options.taskPrefix + 'build-styles');
-      gulp.watch(options.styles.src, ['build-styles']);
+      gulp.watch(options.styles.src + '/**/*.scss', [options.taskPrefix + 'build-styles']);
```
Comment (mandragorn) [inline tasks/watch.js]: odd, for some reason I thought this was already fixed.

File: tasks/build-styles.js [hunk 3]
```diff
@@ -10,16 +10,17 @@ module.exports = function(gulp, options) {
...
+          .on('data', function(data) {
+            log.mark('[SASS] ' + data.contents.length + ' bytes written');
+          })
+          .on('error', function(err) {
             log.error('[SASS] @ ' + currentDateTime());
             log.warn('File: [line:' + err.line + ', col:' + err.column + '] ' + err.file);
             log.warn('Message: ' + err.message);
-          }
-        }))
+            this.emit('end');
```
Comment (mandragorn) [inline tasks/build-styles.js]: emitting 'end' doesn't kill the watch command?

**Comment by mandragorn** [comment]
+1 - just need to make sure the .emit('end') does the correct thing and this can be merged.

File: tasks/build-styles.js (hunk 3 above)
Comment (blakejwc) [inline tasks/build-styles.js]: no, it should just end that pipe process

**Pull request merged**